### PR TITLE
README: first fix - formatting and grammar correction

### PR DESCRIPTION
**Pull request by jeslyw**
## Description

Grammar corrections and better formatting for improved document readability 

## Related Issue

Beautify README.md #36 

## Motivation and Context

[Issue #36 ](https://github.com/SauveJeanLuc/coffee-tasks/issues/36)

First fix including grammar corrections and formatting changes. Open for further improvement :) 

## How Has This Been Tested?

# Checklist:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,43 @@
-# Coffee Tasks
+# Coffee Tasks :coffee:
```
Comment (SauveJeanLuc) [inline README.md:1]: You can remove this coffee emoji.
It's not in sync with the app's logo.

Comment (jeslyw) [reply README.md]: @SauveJeanLuc done :)

**Review by SauveJeanLuc — APPROVED** [review]

**Comment by SauveJeanLuc** [comment]
@jeslyw, consider resolving conflicts

**Comment by jeslyw** [comment]
> @jeslyw, consider resolving conflicts

@SauveJeanLuc should be good now :)

**Comment by SauveJeanLuc** [comment]
@jeslyw ,
Can you resolve the conflicts ?

File: README.md [hunk 2]
```diff
@@ -3,39 +3,40 @@
...
+Originally designed to help beginner front-end devs build a solid
+and real-world working app through the course of - [Hacktoberfest](https://hacktoberfest.com/).
 
-## UI/UX Design
-We have made designs upon which our project is based:
+:exclamation: **All PRs To Be Merged From 1st October 2022** :exclamation:
 
-[Figma Design](https://www.figma.com/file/2DdRHkoszU7lxTvcyBchJ8/React-TODO?node-id=0%3A1)
+## UI/UX Design :art:
+=======
```
Comment (SauveJeanLuc) [inline README.md:26]: What are these for ?

**Pull request closed without merging**